src/schemas/21-526EZ-allclaims/schema.js

```diff
@@ -473,7 +473,8 @@ const schema = {
       enum: ['losingHousing', 'leavingShelter', 'other']
     },
     otherAtRiskHousing: {
-      type: 'string'
+      type: 'string',
+      maxLength: 500,
```

Comment:
I'm assuming that this is the property translated to the FE, whereas the prod error will show the untranslated version. I think this is done [here](https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/evss/disability_compensation_form/data_translation_all_claim.rb#L359).